photon-lib/src/main/java/org/photonvision/simulation/SimCameraProperties.java

```diff
@@ -160,6 +139,94 @@ public SimCameraProperties(Path path, int width, int height) throws IOException
         if (!success) throw new IOException("Requested resolution not found in calibration");
     }
 
+    private void setupCalibration(JsonNode calib, int jsonWidth, int jsonHeight) {
+        // get the relevant calibration values
+        var jsonIntrinsicsNode = calib.get("cameraIntrinsics").get("data");
+        double[] jsonIntrinsics = new double[jsonIntrinsicsNode.size()];
+        for (int j = 0; j < jsonIntrinsicsNode.size(); j++) {
+            jsonIntrinsics[j] = jsonIntrinsicsNode.get(j).asDouble();
+        }
+        var jsonDistortNode = calib.get("distCoeffs").get("data");
+        double[] jsonDistortion = new double[8];
+        Arrays.fill(jsonDistortion, 0);
+        for (int j = 0; j < jsonDistortNode.size(); j++) {
+            jsonDistortion[j] = jsonDistortNode.get(j).asDouble();
+        }
+
+        // assign the read JSON values to this CameraProperties
+        setCalibration(
+                jsonWidth,
+                jsonHeight,
+                MatBuilder.fill(Nat.N3(), Nat.N3(), jsonIntrinsics),
+                MatBuilder.fill(Nat.N8(), Nat.N1(), jsonDistortion));
+    }
+
+    private void setupCalibrationError(JsonNode calib) {
+        var jsonViewErrors = calib.get("perViewErrors");
+        var jsonErrorStdDev = calib.get("standardDeviation");
+        double jsonAvgErrorX = 0;
+        double jsonAvgErrorY = 0;
+        double jsonErrorStdDevX = 0;
+        double jsonErrorStdDevY = 0;
+
+        if (jsonViewErrors != null && jsonErrorStdDev != null) {
```

Comment:
Should I bother with a fallback for the existing logic?

Reply:
Probably unnecessary.

The load-by-path option here has not really been used since the SQLite storage was implemented and a calibration json was no longer easily accessible. Has this changed?

Reply:
There is still currently no longer any way to access a camera config file no. The calibration json file is still easy to get. Let's kill this feature if we don't need it anymore?

Reply:
I've deprecated the two applicable constructors to this feature.

Reply:
Given that we dropped support for this feature from the backend back in 2023, and the ctor is now a no-op, let's just delete them? Or open to your thoughts @amquake

Reply:
Oh, it seems you can export the camera calibration json again now? That would make this constructor useful. It's just to load the resolution/intrinsics/distortion/reprojection error from the exported json instead of copying those values over manually.